packages/community-sdk/src/services/verifyAdmitPass.ts

```diff
@@ -1,16 +1,22 @@
-import { ethers, Signer, BigNumber } from 'ethers';
+import { BigNumber, ethers, Signer } from 'ethers';
 import { getLeavesAndRootFromIpfs } from '../utils/getIpfsLeavesAndRoot';
 import keccak256 from 'keccak256';
-import { getAlphaPassContract } from '@voltz-protocol/commons-v2';
+import {
+  getAlphaPassContract,
+  isTestnet,
+  SupportedChainId,
+} from '@voltz-protocol/commons-v2';
 
 /**
  *
  * @note checks if the user owns an admin pass on-chain
  */
 export async function verifyAdmitPass(owner: Signer): Promise<boolean> {
-  const ownerAddress = await owner.getAddress();
   const chainId = await owner.getChainId();
-
+  if (isTestnet(chainId as SupportedChainId)) {
```

Comment:
Nice utility. What if we turn the function isTestnet(chainId: number) and the check if chainId is of type SupportedChainId is done at the function level? Hence, we avoid this type of casting here.